src/gui-wizard-gtk/wizard.c

```diff
@@ -3487,8 +3496,12 @@ static void add_pages(void)
     g_rb_forbidden_words   = GTK_RADIO_BUTTON( gtk_builder_get_object(g_builder, "rb_forbidden_words"));
     g_rb_custom_search     = GTK_RADIO_BUTTON( gtk_builder_get_object(g_builder, "rb_custom_search"));
     g_exp_search           = GTK_EXPANDER(     gtk_builder_get_object(g_builder, "expander_search"));
+    report_stack = GTK_WIDGET(gtk_builder_get_object(g_builder, "report-stack"));
+    report_status_box = GTK_WIDGET(gtk_builder_get_object(g_builder, "report-status-box"));
+    report_warning_box = GTK_WIDGET(gtk_builder_get_object(g_builder, "report-warning-box"));
     g_spinner_event_log    = GTK_SPINNER(      gtk_builder_get_object(g_builder, "spinner_event_log"));
     g_img_process_fail     = GTK_IMAGE(        gtk_builder_get_object(g_builder, "img_process_fail"));
+    report_warning_label_box = GTK_WIDGET(gtk_builder_get_object(g_builder, "report-warning-label-box"));
```

Comment:
Can you format your code changes so it's consistent with the rest?

Reply:
Sure.